### PR TITLE
chore(trading): handle timeout and offline errors

### LINKED ISSUE #2814: Filtering order list causes a timeout
## Description

When selecting a filter in the Orders tab, a Timeout error can occur. 

## Expected behavior

Orders can be filtered, no timeout. 

## Screenshots

image

## Additional context

Raised by Lewis

### PR DESCRIPTION
**Pull request by macqbat**
# Related issues 🔗

Closes #2814

# Description ℹ️

Add better handle of timeout error, fix some minor bugs in dataProvider

# Demo 📺

image

# Technical 👨‍🔧

Details of technical implementation that reviewers may need to be aware of, if applicable.

### CODE REVIEW

File: libs/accounts/src/lib/accounts-manager.tsx [hunk 1]
```diff
@@ -24,15 +24,20 @@ export const AccountManager = ({
...
 
-  const { data, loading, error } = useDataProvider<AccountFields[], never>({
+  const { data, loading, error, reload } = useDataProvider<
+    AccountFields[],
+    never
+  >({
     dataProvider: aggregatedAccountsDataProvider,
     variables,
   });
+  console.log('data', data);
```
Comment (mattrussell36) [inline libs/accounts/src/lib/accounts-manager.tsx:34]: chore(blocking): remove logs

File: libs/environment/src/components/network-loader/network-loader.tsx [hunk 2]
```diff
@@ -28,6 +29,19 @@ export function NetworkLoader({
     return undefined;
   }, [VEGA_URL, cache]);
 
+  const offlineLayer = useMemo(() => {
```
Comment (mattrussell36) [inline libs/environment/src/components/network-loader/network-loader.tsx:32]: chore(blocking): I'd rather not block the UI if a user loses connection. Can we remove and address in a later sprint.

https://github.com/vegaprotocol/frontend-monorepo/issues/2922

**Review by mattrussell36 — CHANGES_REQUESTED** [review]

File: libs/environment/src/hooks/use-offline-listener.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import { useState, useEffect } from 'react';
+
+export const useOfflineListener = () => {
```
[the remark this answers is not in the document]
Comment (macqbat) [reply libs/environment/src/hooks/use-offline-listener.tsx:3]: sure, I'm gonna remove this one

File: libs/accounts/src/lib/accounts-manager.tsx (hunk 1 above)
Comment (macqbat) [reply libs/accounts/src/lib/accounts-manager.tsx:34]: sure, sorry about that :-)

File: libs/environment/src/components/network-loader/network-loader.tsx (hunk 2 above)
Comment (macqbat) [reply libs/environment/src/components/network-loader/network-loader.tsx]: ok, agree

**Review by jeremyletang — APPROVED** [review]

**Review by mattrussell36 — APPROVED** [review]

**Pull request merged**